user/src/com/google/gwt/logging/client/ConsoleLogHandler.java

```diff
@@ -77,7 +77,9 @@ private native void info(String message) /*-{
     window.console.info(message);
   }-*/;
 
-  private native void log(String message) /*-{
-    window.console.log(message);
+  private native void logVerbose(String message) /*-{
```

Comment:
Please rename the method to `debug` to match the naming scheme of all the other logging methods. Also the comment can be removed as it is already pretty clear what is going on.